Documentation/network/servicemesh/ingress-to-gateway/ingress-to-gateway.rst

```diff
@@ -0,0 +1,109 @@
+
+.. _gs_ingress-to-gateway:
+
+*********************************
+Migrating from Ingress to Gateway
+*********************************
+
+The Gateway API is not only the long-term successor to the Ingress API ; it also supports use cases beyond HTTP/HTTPS-based applications.
+
+This section highlights some of the limitations with Ingress, explains some of the benefits of the Gateway API and describes 
+some of the options available with migrating from Ingress API to Gateway API.
+
+Ingress API Limitations
+#######################
+
+The development of the Gateway API stemmed from the realization that the Kubernetes Ingress API had some limitations.
+
+- Limited support for advanced routing:
+  
+The Ingress API supports basic routing based on path and host rules, but it lacks native support for more advanced routing 
+features such as traffic splitting, header modification, and URL rewriting.
+
+- Limited protocol support:
+  
+The Ingress API only supports HTTP and HTTPS traffic, and does not natively support other protocols such as TCP or UDP.
+The specification of Ingress API was simply too limited and not extensible enough. To address these technical limitations, 
+software vendors and developers created vendor-specific annotations to provide these specific capabilities. But using annotations 
+ended up creating inconsistencies from one Ingress Controller to another. This means that, if you wanted to start using a different 
+Ingress Controller over another, you would likely face some conversion issues as annotations tend to be vendor-specific.
+
+- Operational constraints:
+  
+Finally, the Ingress API suffers from operational constraints: it simply is not well-suited for multi-team clusters with shared load-balancing infrastructure.
+
+Benefits of Gateway API
+#######################
+
+The Gateway API was designed from the ground up to address the Ingress API limitations. The team behind the Gateway API is a Kubernetes SIG-Network project.
+
+.. note::
+
+    You can find out more information about the project on its `website <https://gateway-api.sigs.k8s.io/>`_.
+
+On the technical front, the Gateway API provides a centralized mechanism for managing and enforcing policies for external traffic, 
+including HTTP routing, TLS termination, traffic splitting/weighting, and header modification.
+
+Native support for these features means annotations are no longer needed to provide support for features that are commonly required
+for ingress traffic patterns. This means that Gateway API resources are more portable from one Gateway API implementation to another.
+
+When customization is required, Gateway API provides several flexible models, including specific extension points to enable diverse 
+traffic patterns. As extensions are added, the Gateway API team will keep looking for common denominators and will promote features 
+to the API conformance to avoid going back to some of the chaos seen with extending Ingress API resources.
```

Comment:
Use present tense, use active voice

Common denominators of what? Be specific.

"Promote" in what sense? Promote a feature through to graduation, or "promotion" in the marketing sense to increase the visibility of an existing GA feature? Less marketing jargon. 

Talking about what something isn't for is less helpful than talking about what it _is_ for.
```suggestion
traffic patterns. As the Gateway API team adds extensions, the team looks for common denominators and promotes features of API conformance to maximize the ease of extending Ingress API resources.
```